nuget/lib/BlingFireUtils.cs

```diff
@@ -101,262 +114,270 @@ public static IEnumerable<string> GetWords(string sentence)
-           ref int StartOffsets,
-           ref int EndOffsets,
-           Int32 MaxBuffSize,
-           int UnkId);
+           in byte inUtf8Str,
+           Int32 inUtf8StrLen,
+           ref int tokenIds,
+           ref int startOffsets,
+           ref int endOffsets,
+           Int32 maxBuffSize,
+           int unkId);
+
         public static int TextToIdsWithOffsets(
             UInt64 model,
-            Span<byte> InUtf8Str,
-            Int32 InUtf8StrLen,
-            Span<int> TokenIds,
-            Span<int> StartOffsets,
-            Span<int> EndOffsets,
-            Int32 MaxBuffSize,
-            int UnkId)
+            Span<byte> inUtf8Str,
+            Int32 inUtf8StrLen,
+            Span<int> tokenIds,
+            Span<int> startOffsets,
+            Span<int> endOffsets,
+            Int32 maxBuffSize,
+            int unkId)
         {
             return TextToIdsWithOffsets(
                 model,
-                MemoryMarshal.GetReference(InUtf8Str),
-                InUtf8StrLen,
-                ref MemoryMarshal.GetReference(TokenIds),
-                ref MemoryMarshal.GetReference(StartOffsets),
-                ref MemoryMarshal.GetReference(EndOffsets),
-                MaxBuffSize,
-                UnkId);
+                MemoryMarshal.GetReference(inUtf8Str),
+                inUtf8StrLen,
+                ref MemoryMarshal.GetReference(tokenIds),
+                ref MemoryMarshal.GetReference(startOffsets),
+                ref MemoryMarshal.GetReference(endOffsets),
+                maxBuffSize,
+                unkId);
         }
 
         [DllImport(BlingFireTokDllName)]
         static extern Int32 NormalizeSpaces(
-            in byte InUtf8Str,
-            Int32 InUtf8StrLen,
-            ref byte OutBuff,
-            Int32 MaxBuffSize,
+            in byte inUtf8Str,
+            Int32 inUtf8StrLen,
+            ref byte outBuff,
+            Int32 maxBuffSize,
             Int32 utf32SpaceCode);
+
         public static int NormalizeSpaces(
-            Span<byte> InUtf8Str,
-            Int32 InUtf8StrLen,
-            Span<byte> OutBuff,
-            Int32 MaxBuffSize,
+            Span<byte> inUtf8Str,
+            Int32 inUtf8StrLen,
+            Span<byte> outBuff,
+            Int32 maxBuffSize,
             Int32 utf32SpaceCode)
         {
             return NormalizeSpaces(
-                MemoryMarshal.GetReference(InUtf8Str),
-                InUtf8StrLen,
-                ref MemoryMarshal.GetReference(OutBuff),
-                MaxBuffSize,
+                MemoryMarshal.GetReference(inUtf8Str),
+                inUtf8StrLen,
+                ref MemoryMarshal.GetReference(outBuff),
+                maxBuffSize,
                 utf32SpaceCode);
         }
 
-        private static char[] _justNewLineChar = new char[] { '\n' };
-        private static char[] _justSpaceChar = new char[] { ' ' };
+        private static char[] justNewLineChar = new char[] { '\n' };

```

Comment:
Please rename it into g_justNewLineChar and g_justSpaceChar so it is clear it is static.